Source/Transmitters/LMCAntennaSignalTransmitter.cc

```diff
@@ -77,7 +77,12 @@ namespace locust
     
     bool AntennaSignalTransmitter::SetAntennaType( std::string transmitterAntennaType )
      {
-     	if (transmitterAntennaType == "antenna-signal-dipole") fAntennaType = 0; // default
+     	if (transmitterAntennaType == "antenna-signal-dipole")
+     		{
+     		fAntennaType = 0; // default
+    		DipoleAntenna* modelTransmitterHardware = new DipoleAntenna;

```

Comment:
Couldn't these two lines be combined into a single line like this ?
 fTransmitterHardware  = new DipoleAntenna;
since TransmitterHardware is a parent class of  DipoleAntenna